spec/workers/pull_request_monitor_handlers/merge_target_titler_spec.rb

```diff
@@ -0,0 +1,52 @@
+describe PullRequestMonitorHandlers::MergeTargetTitler do
+  let(:branch)         { create(:pr_branch) }
+  let(:github_service) { stub_github_service }
+
+  before do
+    stub_sidekiq_logger
+    stub_settings(:merge_target_titler => {:enabled_repos => [branch.repo.name]})
+  end
+
+  context "when the branch has a non-master merge target" do
+    before do
+      branch.update_attributes!(:merge_target => "darga")
+    end
+
+    it "and not already titled" do
+      expect(github_service).to receive(:user).and_return("SomeUser")
```

Comment:
Yeah, for some reason the github api requires that I "resend" the user and repo...it's not being honored from the higher level calls.  It feels like a bug in the gem.